### ISSUE TITLE
ww12.vidstreaming.io

### ISSUE THREAD

**Issue by FeeninForMore** [hagezi/dns-blocklists#2827]
### Which AdBlocker/DNS cloud service do you use?

AdGuard DNS

### ControlD users

- [ ] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [ ] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Multi PRO

### Which domain(s) should be unblocked?

ww12.vidstreaming.io

### Why should the domain(s) be unblocked?

Vidstreaming is a anime website where you can watch anime, I don't think it should be blocked

**Comment by FeeninForMore** [comment]
It's also a server multiple steaming websites use

**Comment by hagezi** [comment]
@FeeninForMore not blocked in normal lists. Only blocked in anti piracy list.

**Comment by FeeninForMore** [comment]
So is the anti piracy list included in the tif? Because I'm not using any anti piracy list

**Comment by hagezi** [comment]
@FeeninForMore No, `ww12.vidstreaming.io` is a parked domain (CNAME  `084725.parkingcrew.net`) and `*.parkingcrew.net` is blocked in normal lists.

![grafik](https://github.com/hagezi/dns-blocklists/assets/104343908/a6d34bf8-84f0-4064-ba6e-5ee6c1eaed89)

![grafik](https://github.com/hagezi/dns-blocklists/assets/104343908/8ad2d23d-ffbf-480b-83e2-b4f0b1f0aef0)
